IoTCoreDefaultApp/IoTCoreDefaultApp/Strings/en-US/Resources.resw

```diff
@@ -318,7 +318,7 @@
     <value>The first, and most basic thing a board should do is blink.</value>
   </data>
   <data name="TutorialBlinkyBody2Text" xml:space="preserve">
-    <value>As mentioned in the "Get Started" section, all the source code for this open source and available on www. windowsondevices.com (look for the IoTCoreDefaultApp).</value>
+    <value>As mentioned in the "Get Started" section, all the source code for this open source are available on www.windowsondevices.com (look for the IoTCoreDefaultApp).</value>
```

Comment:
Great catch ... thank you!  The wording we intended was: "all of the source code for this is open source and is available on"